vendor/github.com/Scalingo/go-scalingo/v6/apps.go

```diff
@@ -275,6 +275,21 @@ func (c *Client) AppsContainersPs(ctx context.Context, app string) ([]Container,
 	return containersRes.Containers, nil
 }
 
+func (c *Client) AppsContainersSendSignal(ctx context.Context, app string, signal string, container string) error {
```

Comment:
You modified a vendored dependency. You should update go-scalingo and release a new version of go-scalingo.

Reply:
Oups, I will thx